pkg/cmd/controller/apiserver_handlers.go

```diff
@@ -16,9 +18,9 @@ func newBypassAuthorizer(auth kauthorizer.Authorizer, paths ...string) kauthoriz
 	return bypassAuthorizer{paths: sets.NewString(paths...), authorizer: auth}
 }
 
-func (a bypassAuthorizer) Authorize(attributes kauthorizer.Attributes) (allowed kauthorizer.Decision, reason string, err error) {
+func (a bypassAuthorizer) Authorize(ctx context.Context, attributes kauthorizer.Attributes) (allowed kauthorizer.Decision, reason string, err error) {
```

Comment:
/cc @deads2k

Reply:
fyi @adambkaplan this change was needed as a result of an upstream change in k8s apiserver

see https://github.com/kubernetes/apiserver/blob/kubernetes-1.17.0-rc.1/pkg/authorization/authorizer/interfaces.go#L71